public/src/js/modules/grid.js

```diff
@@ -4,7 +4,7 @@ import {CONST} from 'modules/vars';
 export function recordUsage(mediaId, frontId) {
     const usageData = {
         mediaId: mediaId,
-        front: frontId,
```

Comment:
... did our linting get stricter, or... ?

Reply:
How strange, I don't think we've changed anything in the old app.